doc/classes/EditorSettings.xml

```diff
@@ -1123,13 +1251,31 @@
 		<member name="text_editor/help/show_help_index" type="bool" setter="" getter="">
 			If [code]true[/code], displays a table of contents at the left of the editor help (at the location where the members overview would appear when editing a script).
 		</member>
+		<member name="text_editor/help/sort_functions_alphabetically" type="bool" setter="" getter="">
+			If [code]true[/code], the script's method list in the Script Editor is sorted alphabetically.
+		</member>
+		<member name="text_editor/script_list/group_help_pages" type="bool" setter="" getter="">
+			If [code]true[/code], class reference pages are grouped together at the bottom of the Script Editor's script list.
+		</member>
+		<member name="text_editor/script_list/list_script_names_as" type="int" setter="" getter="">
+			Specifies how scripts should be displayed in Script Editor's script list. If using the "Name" option, the scripts can still show part of their paths if their names are conflicting.
```

Comment:
```suggestion
			Specifies how script paths should be displayed in Script Editor's script list. If using the "Name" option and some scripts share the same file name, part of their paths are revealed to avoid conflicts.
```